tests/unit_test/utility_test.py

```
@@ -838,6 +838,63 @@ async def test_handle_verification(self, validate_and_send_mail_mock):
         )
 
+    @pytest.mark.asyncio
+    @patch("kairon.shared.utils.MailUtility.validate_and_send_mail", autospec=True)
+    async def test_handle_verification_with_valid_first_name(self, validate_and_send_mail_mock):
+        mail_type = "verification"
+        email = "sampletest@gmail.com"
+        first_name = "Sample"  # Valid first name
+        base_url = Utility.environment["app"]["frontend_url"]
+
+        Utility.email_conf["email"]["templates"]["verification"] = (
+            open("template/emails/verification.html", "rb").read().decode()
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)